chaos_wrapper.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+function rollback() {
+    git checkout master
+    git checkout HEAD^
```

Comment:
Actually, it occurs to me that the use of `git checkout master` is completely nonsensical here because then we will never get past the second-to-last commit :face_with_head_bandage: